File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/workout/model/Workout.kt
package pt.isel.leic.ptgest.domain.workout.model

import pt.isel.leic.ptgest.domain.workout.MuscleGroup

data class Workout(
    val id: Int,
    val name: String,
    val description: String?,
    val muscleGroup: List<MuscleGroup>
)

File: code/js/src/views/user/trainerViews/models/workouts/Workout.ts
class Workout {
    id: number
    name: string
    description: string | null
    muscleGroup: string[]
    isFavorite: boolean

    constructor(id: number, name: string, description: string | null, muscleGroup: string[], isFavorite: boolean) {
        this.id = id
        this.name = name
        this.description = description
        this.muscleGroup = muscleGroup
        this.isFavorite = isFavorite
    }
}
export default Workout
